spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -0,0 +1,81 @@
+from typing import Iterable, Union
+import numpy as np
+from scipy.stats import norm
+from spikeinterface.core import BaseRecording, BaseRecordingSegment, get_chunk_with_margin
+from spikeinterface.core.core_tools import define_function_from_class
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+
+
+class GaussianFilterRecording(BasePreprocessor):
+	"""
+	Class for performing gaussian filtering/smoothing on a recording.
```

Comment:
@DradeAW I think this has the usual formatting issue

Reply:
What do you mean?

Reply:
Oh you mean tabs and spaces!

I'll get right on it ;)